clients/fides-js/src/lib/initialize.ts

```diff
@@ -328,15 +336,23 @@ export const initialize = async ({
         fidesRegionString,
         options.fidesApiUrl,
         options.debug,
-        options.apiOptions,
-        cookie.identity.fides_user_device_id
+        options.apiOptions
       );
     }
 
     if (
       isPrivacyExperience(effectiveExperience) &&
       experienceIsValid(effectiveExperience, options)
     ) {
+      // apply GPC preferences
+      if (shouldInitOverlay && isPrivacyExperience(effectiveExperience)) {
+        automaticallyApplyGPCPreferences({
+          cookie,
+          fidesRegionString,
+          effectiveExperience,
+          fidesOptions: options,
+        });
+      }
       const updated = await updateCookieAndExperience({
```

Comment:
we can probably rename the `updateCookieAndExperience` function now? it doesn't update the cookie anymore, right? (and arguably doesn't need to update the experience anymore either, but we can punt on that if that makes more sense)

Reply:
good point!